app/database/result_store.go

```diff
@@ -28,7 +28,7 @@ func (s *ResultStore) GetHintsInfoForActiveAttempt(participantID, attemptID, ite
 	var hintsInfo HintsInfo
 	mustNotBeError(s.Results().
 		ByID(participantID, attemptID, itemID).
-		WithExclusiveWriteLock().Select("hints_requested, hints_cached").
```

Comment:
So the `.Select("hints_requested, hints_cached")` has disappeared?

Reply:
Wow! Nice catch! Fixed.